bitnami/schema-registry/README.md

```diff
@@ -108,6 +108,34 @@ Please check the configuration settings for the Kakfa service in the [Kafka's RE
 
 Please check the configuration settings for the Kakfa service in the [Zookeeper's README file](https://github.com/bitnami/containers/tree/main/bitnami/zookeeper#configuration).
 
+## Security
```

Comment:
In other READMEs, the security section is typically nested within the `Configuration` section. Do you find it appropriate to maintain this structure?
```suggestion
### Security
```

Reply:
Sure, if it is consistent with the rest of the documentation then it makes perfect sense